tests/integ/sagemaker/serve/test_serve_js_happy.py

```diff
@@ -46,6 +48,17 @@ def happy_model_builder(sagemaker_session):
     )
 
 
+@pytest.fixture
+def happy_model_builder_gated_model(sagemaker_session):
```

Comment:
Can we add a test for a non jumpstart model + has keyword private + wants to test in local mode? We should let such a test case pass through.

Reply:
Done